code/modules/projectiles/guns/projectile/shotgun.dm

```diff
@@ -103,23 +126,60 @@
 	return 0
 
 /obj/item/weapon/gun/projectile/shotgun/doublebarrel/attack_self(mob/living/user as mob)
+	if(!wielded)
+		wield(user)
+		src.update_wield(user)
+	else
+		unwield(user)
+
+/obj/item/weapon/gun/projectile/shotgun/doublebarrel/update_wield(mob/user)
+	..()
+	item_state = "dshotgun[wielded ? 1 : 0]"
+	if(user)
+		user.update_inv_hands()
+
+/obj/item/weapon/gun/projectile/shotgun/doublebarrel/AltClick()
+	if(!broke)
+		var/i = 0
+		if(clumsy_check(usr))
```

Comment:
if(clumsy_check && clumsy_check(usr)) so varediting it can get around clumsiness